activesupport/lib/active_support/dependencies.rb

```diff
@@ -92,10 +92,17 @@ def initialize
         @stack = Hash.new { |h,k| h[k] = [] }
       end
 
+      # Calls the given block once for each element in stack, passing that
+      # element as a parameter. An Enumerator is returned if no block is given.
+      #
+      #   new_constants = ActiveSupport::Dependencies::WatchStack.new.new_constants
+      #   new_constants.each { |c| puts c }
+      #   # => RaisesNoMethodError
```

Comment:
Is this really useful? It seems like you're just explaining a standard each loop? Which I think we should assume people know how that works.

I don't understand you example either - why does it raise a NoMethodError?